src/renderer/page/rewards/view.jsx

```diff
@@ -9,9 +9,11 @@ import classnames from 'classnames';
 
 type Props = {
   doAuth: () => void,
+  fetchRewards: () => void,
   navigate: string => void,
   fetching: boolean,
   rewards: Array<{ reward_type: boolean }>,
+  claimed: {},
```

Comment:
Could you add a new type in `types/reward`? Then we can improve the above two lines with actual types.